utils/s2n_init.c

```diff
@@ -42,6 +42,7 @@ int s2n_init(void)
     GUARD_POSIX(s2n_mem_init());
     GUARD_AS_POSIX(s2n_rand_init());
     GUARD_POSIX(s2n_cipher_suites_init());
+    GUARD_POSIX(s2n_ecc_evp_init());
```

Comment:
This has a dependency on ordering. s2n_fips_init() must be called first. Might be worth a comment here